packages/extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@logion/extension",
-  "version": "0.7.4",
+  "version": "0.7.5-1",
```

Comment:
Maybe we should make `client` dependency a peer dependency?

Reply:
I'll see that later in the sprint.